### PR TITLE
Finish implementation for property directory

### LINKED ISSUE #45: Property directory not fully created

### PR DESCRIPTION
**Pull request by lkwlkww**
close #45

### CODE REVIEW

**Comment by misterpuffin** [comment]
can help to resolve the checkstyle errors? thanks!

File: src/main/java/seedu/condonery/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -2,59 +2,53 @@
...
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_NAME + "John Doe "
-            + PREFIX_PHONE + "98765432 "
-            + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_NAME + "PINNACLE@DUXTON "
```
Comment (misterpuffin) [inline src/main/java/seedu/condonery/logic/commands/AddCommand.java:25]: iirc, the @ symbol is not a valid character in the Name regex for Property. Might have to check if it's possible to add property with such a name!

File: src/main/java/seedu/condonery/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -2,59 +2,53 @@
...
     }
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
-        if (model.hasPerson(toAdd)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PERSON);
+        if (model.hasProperty(toAdd)) {
+            throw new CommandException(MESSAGE_DUPLICATE_property);
```
Comment (misterpuffin) [inline src/main/java/seedu/condonery/logic/commands/AddCommand.java:48]: might want to capitalize this to keep it consistent!

**Review by misterpuffin — CHANGES_REQUESTED** [review]
Overall, thanks for the great work! Thanks for doing the delete/edit commands as well. Just small comments but otherwise lgtm.

File: src/main/java/seedu/condonery/logic/commands/AddCommand.java (hunk 2 above)
Comment (lkwlkww) [reply src/main/java/seedu/condonery/logic/commands/AddCommand.java]: Done, thanks!

**Review by yuehernkang — APPROVED** [review]
LGTM

**Review by misterpuffin — APPROVED** [review]
lgtm

**Pull request merged**